sources/packages/web/src/views/supporting-user/Login.vue

```diff
@@ -1,41 +1,46 @@
 <template>
-  <v-card elevation="2" class="mx-auto mt-15" max-width="690px" outlined>
-    <v-card-header>
-      <v-card-header-text>
-        <v-card-title class="my-6 bold-text" style="font-size: 27px"
-          >Welcome to Student Application Supporting Information
-          Portal</v-card-title
-        >
-        <v-card-subtitle>Welcome text goes here…</v-card-subtitle>
-      </v-card-header-text>
-    </v-card-header>
-    <v-card-text
-      >We are using BCSC for authentication. Please click on Login/Register
-      buttons below to start your sign in/sign up.</v-card-text
-    >
-    <v-row class="m-3">
-      <v-col>
-        <v-btn
-          justify-center
-          color="primary"
-          @click="login"
-          class="float-right"
-          prepend-icon="fa:fa fa-user"
-        >
-          Login with BCSC
-        </v-btn>
-      </v-col>
-      <v-col>
-        <v-btn
-          color="primary"
-          @click="login"
-          prepend-icon="mdi-account-star-outline"
-        >
-          Sign Up with BCSC
-        </v-btn>
-      </v-col>
-    </v-row>
-  </v-card>
+  <v-card elevation="2" class="mx-auto mt-15" max-width="730px">
+    <v-card-text>
+      <v-row
+        ><v-col cols="15"
```

Comment:
mm..cols options are from 1 to 12. also you can use `md` instead of `cols`

Reply:
yeah, sorry. That came from another login page we have here. Thanks!

Reply:
oh is it? from which login.vue @andrepestana-aot

Reply:
aest Login page

Reply:
@guru-aot is aest login page part of your PR or your previous PR. we need some refactoring in login page

Reply:
Its was part of my previous story. I am not changing anything now.